From c1c0bc827fabea1c73b7dee71b90e7cdef5c9290 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 20 Dec 2023 15:56:32 +0100
Subject: [PATCH 0861/1141] ARM: dts: stm32: add support of continuous display
 on stm32mp157 DK2 boards

Add DSI clocks.

Change-Id: If310647a7720ac21ab79323e4ec7a64536f2235e
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/352879
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm/boot/dts/stm32mp157c-dk2.dts | 3 ++-
 arch/arm/boot/dts/stm32mp157f-dk2.dts | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157c-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157c-dk2.dts
@@ -31,7 +31,7 @@
 
 		framebuffer {
 			compatible = "simple-framebuffer";
-			clocks = <&rcc LTDC_PX>;
+			clocks = <&rcc LTDC_PX>, <&rcc DSI>, <&rcc DSI_PX>;
 			status = "disabled";
 		};
 	};
@@ -102,6 +102,7 @@
 };
 
 &ltdc {
+	default-on;
 	status = "okay";
 
 	port {
--- a/arch/arm/boot/dts/stm32mp157f-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2.dts
@@ -30,7 +30,7 @@
 
 		framebuffer {
 			compatible = "simple-framebuffer";
-			clocks = <&rcc LTDC_PX>;
+			clocks = <&rcc LTDC_PX>, <&rcc DSI>, <&rcc DSI_PX>;
 			status = "disabled";
 		};
 	};
@@ -101,6 +101,7 @@
 };
 
 &ltdc {
+	default-on;
 	status = "okay";
 
 	port {
